test/integration/editors/triggered/CommunicationMappingPlugin.test.ts

```diff
@@ -0,0 +1,192 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import CommunicationMappingPlugin from '../../../../src/triggered/CommunicationMapping.js';
+import { List } from '@material/mwc-list';
+import { MockWizardEditor } from '../../../mock-wizard-editor.js';
+
+describe('CommunicationMappingPlugin', () => {
+  let doc: Document;
+  customElements.define(
+    'communication-mapping-plugin',
+    CommunicationMappingPlugin
+  );
+  let parent: MockWizardEditor;
+  let element: CommunicationMappingPlugin;
+
+  beforeEach(async () => {
+    parent = await fixture(
+      html`<mock-wizard-editor
+        ><communication-mapping-plugin></communication-mapping-plugin
+      ></mock-wizard-editor>`
+    );
+
+    element = <CommunicationMappingPlugin>(
+      parent.querySelector('communication-mapping-plugin')!
+    );
+
+    doc = await fetch('public/xml/testfiles/comm-map.scd')
+      .then(response => response.text())
+      .then(str => new DOMParser().parseFromString(str, 'application/xml'));
+    element.doc = doc;
+    await element.updateComplete;
+  });
+
+  describe('communication mapping wizard', () => {
+    beforeEach(async () => {
+      await element.trigger();
+      await parent.updateComplete;
+    });
+
+    it('opens the communication mapping wizard on trigger', async () => {
+      expect(parent.wizardUI.dialogs.length).to.equal(1);
+    });
+
+    it('closes on secondary action', async () => {
+      (<HTMLElement>(
+        parent.wizardUI.dialog!.querySelector(
+          'mwc-button[slot="secondaryAction"]'
+        )
+      )).click();
+      await parent.updateComplete;
+      await new Promise(resolve => setTimeout(resolve, 100)); // await animation
+      expect(parent.wizardUI.dialogs.length).to.equal(0);
+    });
+
+    it('groups connections to unique pair of source ied, sink ied and control block', () => {
+      expect(
+        parent.wizardUI.dialog?.querySelectorAll('mwc-list-item').length
+      ).to.equal(4);
+    });
+
+    it('indicates the control block type with mwc-list graphic slot', () => {
+      expect(
+        parent.wizardUI.dialog!.querySelectorAll('mwc-list-item > mwc-icon')
+          .length
+      ).to.equal(4);
+    });
+
+    it('show the source ied, sink ied and control block', () => {
+      expect(
+        parent.wizardUI.dialog!.querySelectorAll('mwc-list-item > mwc-icon')
+          .length
+      ).to.equal(4);
+    });
+  });
+
+  describe('control block connection wizard', () => {
+    let commMappings: List;
+    beforeEach(async () => {
+      await element.trigger();
+      await parent.updateComplete;
+      await new Promise(resolve => setTimeout(resolve, 100)); // await animation
+      commMappings = <List>(
+        parent.wizardUI.dialog?.querySelector('filtered-list')
+      );
+    });
+
+    it('opens on Connection item click', async () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
    it("opens on Connection item click", async () => {
```